src/test/java/seedu/exercise/logic/commands/CustomAddCommandTest.java

```diff
@@ -26,58 +27,53 @@
 import seedu.exercise.model.ReadOnlyResourceBook;
 import seedu.exercise.model.UserPrefs;
 import seedu.exercise.model.property.CustomProperty;
+import seedu.exercise.model.property.PropertyBook;
 import seedu.exercise.testutil.builder.CustomPropertyBuilder;
 
 class CustomAddCommandTest {
 
     private Model model = new ModelManager(getTypicalExerciseBook(), new ReadOnlyResourceBook<>(),
-            new ReadOnlyResourceBook<>(), new ReadOnlyResourceBook<>(), new UserPrefs(), getDefaultPropertyBook());
+        new ReadOnlyResourceBook<>(), new ReadOnlyResourceBook<>(), new UserPrefs());
+    private PropertyBook propertyBook = PropertyBook.getInstance();
 
     @BeforeEach
     public void reset() {
```

Comment:
standard naming for `@BeforeEach` annotated methods is `setUp()`

Reply:
Noted and changed.